src/components/dashboard/commonConsentSections/FirstCommonConsentSection.tsx

```diff
@@ -233,23 +289,43 @@ function FirstCommonConsentSection({
         <ResponsiveStepWrapper>
           <ProgressBar step={step} maxSteps={maxSteps} />
           <div className="text-step-wrapper">
-            <Network />
-            <div className="header-wrapper">
-              <h1>{t('form.firstCommonConsent.transferAndRights')}</h1>
+            <div className="icon-wrapper">
+              <Network />
             </div>
-            <p>{t('form.firstCommonConsent.transferAway')}</p>{' '}
-            <p>{t('form.firstCommonConsent.transferToUS')}</p>
-            <h2>{t('form.firstCommonConsent.yourRights')}</h2>
+            <Typography variant="h3" className="common-title-spaced">
+              {t('form.firstCommonConsent.transferAndRights')}
+            </Typography>
+
+            <Typography variant="body2" className="bottom-space">
+              {t('form.firstCommonConsent.transferAway')}
+            </Typography>
+
+            <Typography variant="body2" className="bottom-space">
```

Comment:
body2 bottom spacing is different in many places, i came up with a specifc bottom-space rule to apply margin-bottom as needed.